lib/enum.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+/**
+ * Turn an enum into a string suitable for debugging.
+ *
+ * @param object group
+ *   Enum group, e.g. bacnet.enum.ConfirmedServiceChoice.
+ *
+ * @param Number value
+ *   Enum value, e.g. 1.  Note that this *must* be an integer value, so you may
+ *   need to use parseInt().  Non-integer values will result in an exception.
+ *
+ * @example
+ * const s = bacnet.enum.getEnumName(
+ *   bacnet.enum.PropertyIdentifier,
+ *   bacnet.enum.PropertyIdentifier.PRESENT_VALUE
+ * );
+ * console.log(s); // "PRESENT_VALUE(85)"
+ */
+module.exports.getEnumName = function(group, value) {
```

Comment:
The enum file should only contain type definitions and no utility functions.

Reply:
Where would you suggest putting the utility function?  I had it in `client.js` originally but since it's only related to enums I thought `enum.js` would be a better fit.

I am using `enum.js` in another module so it is nice that I can get the enum utility function without including any additional node-bacstack code.

Reply:
Actually this is part of PR #134 instead, so if you want to resolve this issue over there then I can rebase this one to pick up the fix.